OpenProblemLibrary/ma117DB/set5/srw2_9_63.pg

```diff
@@ -20,35 +20,37 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
+  "PGML.pl",
+  "MathObjects.pl",
   "PGcourse.pl"
 );
 
-TEXT(beginproblem());
 $showPartialCorrectAnswers = 1;
 
+Context("Numeric");
+
 $a = random(2,9,1);
+$ans1 = Compute("-$a");
+$ans2 = Infinity;
+$ans3 = Formula("sqrt(x)-$a");
+
+BEGIN_PGML
+The function [`f(x)=(x+[$a])^2`] is not one-to-one. Choose a largest possible
+domain containing the number 100 so that the function restricted to the domain is one-to-one.
+
+The largest possible domain is [`\left[\strut\right.`][________]{$ans1},[________]{$ans2}[`\left.\strut\right)`].
+
+The inverse function is [`g(x)=`][_______________]{$ans3}.
```

Comment:
This should specify that it is asking for the inverse function on the domain given above.  How about "If x is restricted to the above domain, then the inverse function is "?

Reply:
Sure, that is definitely better.